src/facade/reply_builder.h

```diff
@@ -12,6 +12,12 @@
 #include "io/io.h"
 
 namespace facade {
+// some constants, used in reply_builder.cc; can be used elsewhere.
+namespace constants {
```

Comment:
we do not do the changes "for future" :)

please bring back.

Reply:
Oh, I use one of the constants: `kSimplePref`, to check if replication succeeded (in `Replicate()`), although this is not really needed anymore. Should I still change it back?